CHANGELOG.md

```diff
@@ -14,42 +14,61 @@
-* `vcd_org_vdc` includes a property `enable_nsxv_distributed_firewall` to enable or disable a NSX-V distributed firewall [[#988](https://github.com/vmware/terraform-provider-vcd/pull/988))
+  backed entities so that it can be inherited by direct Org VDC networks ([#984](https://github.com/vmware/terraform-provider-vcd/pull/984))
+* `vcd_org_vdc` includes a property `enable_nsxv_distributed_firewall` to enable or disable a NSX-V distributed firewall ([#988](https://github.com/vmware/terraform-provider-vcd/pull/988))
 * `vcd_nsxt_edgegateway` resource and data source got automatic IP allocation support using new
   configuration fields `subnet_with_total_ip_count`, `subnet_with_ip_count` and `total_allocated_ip_count` fields ([#991](https://github.com/vmware/terraform-provider-vcd/pull/991))
 * `vcd_nsxt_edgegateway` resource and data source expose `used_ip_count` and `unused_ip_count`
-  attributes ([#991](https://github.com/vmware/terraform-provider-vcd/pull/991))
+  attributes ([#991](https://github.com/vmware/terraform-provider-vcd/pull/991), [#1047](https://github.com/vmware/terraform-provider-vcd/pull/1047))
 * `vcd_nsxt_alb_settings` resource and data source adds two new fields `is_transparent_mode_enabled`
   and `ipv6_service_network_specification` ([#996](https://github.com/vmware/terraform-provider-vcd/pull/996))
 * Resources `vcd_vapp_network` and `vcd_vapp_org_network` add convenience flag
   `reboot_vapp_on_removal`. When enabled, it will power off parent vApp (and power back on after
   if it was before) during vApp network removal. This improves workflows with VCD 10.4.1+ which
   returns an error when removing vApp networks from powered on vApps ([#1004](https://github.com/vmware/terraform-provider-vcd/pull/1004))
+* `vcd_vapp_vm` and `vcd_vm` resources support security tag management via new field `security_tags` ([#1006](https://github.com/vmware/terraform-provider-vcd/pull/1006), [#1046](https://github.com/vmware/terraform-provider-vcd/pull/1046))
+* Resource `vcd_nsxt_ipsec_vpn_tunnel` adds support for custom `remote_id` field and certificate
+  based auth via fields `authentication_mode`, `certificate_id`, `ca_certificate_id` ([#1010](https://github.com/vmware/terraform-provider-vcd/pull/1010))
 * `vcd_org_group` adds `OAUTH` as an option to argument `provider_type` ([#1013](https://github.com/vmware/terraform-provider-vcd/pull/1013))
+* Resource and data source `vcd_nsxt_alb_virtual_service` add support for Transparent mode in VCD
+  10.4.1+ via field `is_transparent_mode_enabled` ([#1024](https://github.com/vmware/terraform-provider-vcd/pull/1024))
+* Resource and data source `vcd_nsxt_alb_pool` add support for Pool Group Membership via field
+  `member_group_id` ([#1024](https://github.com/vmware/terraform-provider-vcd/pull/1024))
+* Support provider authentication using Active Service Accounts ([#1040](https://github.com/vmware/terraform-provider-vcd/pull/1040))
```

Comment:
Let's move this feature one line below, so it becomes more visible as the last item in the list and separate from all the NSX-T features above.

Reply:
done